index.js

```diff
@@ -108,10 +108,18 @@ module.exports = (helpText, options) => {
 	delete argv._;
 
 	const flags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
+	const unnormalizedFlags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
```

Comment:
It's wasteful to run the exact same function twice. You can just copy the `flags` object using object spread.

Reply:
Didn't know that existed, thanks for teaching me something.
Could we do it this way ? 
```Javascript
	const flags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
	const unnormalizedFlags = {...flags};
```